### DISCUSSION TITLE
nb Converter did not convert all the notebook

### DISCUSSION THREAD

**Discussion by sorobedio** [seominjoon/kaist-ai605#91] [Q&A] [answered]
Hello TA i just realize that that my submitted pdf is not all the notebook. what to do? will i be penalized for converting later

**Comment by miyoungko** [comment, marked as answer]
Hi, please send the entire pdf file by mail (send it to: miyoungko@kaist.ac.kr).
If your pdf result is same as the colab or .ipynb file, there will be no penalize for your submission.
